dm/loader/lightning.go

```diff
@@ -299,11 +293,8 @@ func (l *LightningLoader) Process(ctx context.Context, pr chan pb.ProcessResult)
 		isCanceled = true
 	default:
 	}
+	pr <- pb.ProcessResult{IsCanceled: isCanceled, Errors: errs}
```

Comment:
why move this code? when `pr` is full (though not possible now) the original log can help us debug it.